### ISSUE TITLE
anonymous view of place 560221 throws error

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#149]
Displays fine for admin user:

http://pleiades.stoa.org/places/560221

anonymous sees e.g., The error has been logged as entry number 1464610371.180.29930496417.

**Comment by davisagli** [comment]
```

Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module Shared.DC.Scripts.Bindings, line 322, in __call__
  Module Shared.DC.Scripts.Bindings, line 359, in _bindAndExec
  Module Products.CMFCore.FSPageTemplate, line 237, in _exec
  Module Products.CMFCore.FSPageTemplate, line 177, in pt_render
  Module Products.PageTemplates.PageTemplate, line 87, in pt_render
  Module zope.pagetemplate.pagetemplate, line 132, in pt_render
   - Warning: Macro expansion failed
   - Warning: <type 'exceptions.KeyError'>: 'macro'
  Module zope.pagetemplate.pagetemplate, line 240, in __call__
  Module collective.newrelic.patches.talinterpreter, line 17, in monkeypatch
  Module newrelic.api.function_trace, line 110, in literal_wrapper
  Module zope.tal.talinterpreter, line 271, in __call__
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 888, in do_useMacro
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 533, in do_optTag_tal
  Module zope.tal.talinterpreter, line 518, in do_optTag
  Module zope.tal.talinterpreter, line 513, in no_tag
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 954, in do_defineSlot
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 533, in do_optTag_tal
  Module zope.tal.talinterpreter, line 518, in do_optTag
  Module zope.tal.talinterpreter, line 513, in no_tag
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 858, in do_defineMacro
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 954, in do_defineSlot
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 533, in do_optTag_tal
  Module zope.tal.talinterpreter, line 518, in do_optTag
  Module zope.tal.talinterpreter, line 513, in no_tag
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 954, in do_defineSlot
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 533, in do_optTag_tal
  Module zope.tal.talinterpreter, line 518, in do_optTag
  Module zope.tal.talinterpreter, line 513, in no_tag
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 946, in do_defineSlot
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 533, in do_optTag_tal
  Module zope.tal.talinterpreter, line 518, in do_optTag
  Module zope.tal.talinterpreter, line 513, in no_tag
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 858, in do_defineMacro
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 533, in do_optTag_tal
  Module zope.tal.talinterpreter, line 518, in do_optTag
  Module zope.tal.talinterpreter, line 513, in no_tag
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 888, in do_useMacro
  Module zope.tal.talinterpreter, line 343, in interpret
  Module zope.tal.talinterpreter, line 583, in do_setLocal_tal
  Module zope.tales.tales, line 696, in evaluate
   - URL: file:/srv/python27-apps/pleiades4/src/Products.PleiadesEntity/Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt
   - Line 70, Column 4
   - Expression: <PathExpr standard:u'context/@@where'>
   - Names:
      {'container': <Place at /plone/places/560221>,
       'context': <Place at /plone/places/560221>,
       'default': <object object at 0x7fd009978bb0>,
       'here': <Place at /plone/places/560221>,
       'loop': {},
       'nothing': None,
       'options': {'args': ()},
       'repeat': <Products.PageTemplates.Expressions.SafeMapping object at 0x7fcfd0e7aa48>,
       'request': <HTTPRequest, URL=http://pleiades.stoa.org/places/560221/base_view>,
       'root': <Application at >,
       'template': <FSPageTemplate at /plone/places/560221/base_view>,
       'traverse_subpath': [],
       'user': <SpecialUser 'Anonymous User'>}
  Module zope.tales.expressions, line 217, in __call__
  Module Products.PageTemplates.Expressions, line 155, in _eval
  Module Products.PageTemplates.Expressions, line 117, in render
  Module pleiades.geographer.views, line 9, in __call__
  Module pleiades.geographer.geo, line 558, in coords
  Module plone.memoize.instance, line 51, in memogetter
  Module pleiades.geographer.geo, line 526, in reprPoint
  Module pleiades.geographer.geo, line 414, in preciseExtents
  Module pleiades.geographer.geo, line 413, in <lambda>
  Module pleiades.geographer.geo, line 489, in precision
  Module plone.memoize.instance, line 51, in memogetter
  Module pleiades.geographer.geo, line 467, in reprExtent
  Module pleiades.geographer.geo, line 400, in gridGeoms
  Module pleiades.geographer.geo, line 73, in __init__
  Module pleiades.capgrids, line 143, in bounds
  Module pleiades.capgrids, line 65, in box
IndexError: No gridsquare c5 in map 60
```

**Comment by davisagli** [comment]
Fixed by the change I made for #143 -- see http://pleiades.jazkarta.com/places/560221

**Comment by paregorios** [comment]
nice
